src/universal/containers/Action/ActionContainer.js

```diff
@@ -1,39 +1,43 @@
 import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
 import Action from 'universal/components/Action/Action';
 import {injectStyleOnce} from 'aphrodite-local-styles/lib/inject';
 import injectGlobals from 'universal/styles/hepha';
 import globalStyles from 'universal/styles/theme/globalStyles';
+import {segmentEventPage} from 'universal/redux/segmentActions';
 
-const updateAnalyticsPage = (lastPage, nextPage) => {
-  if (typeof window !== 'undefined' &&
-    typeof window.analytics !== 'undefined') {
-    window.analytics.page('', nextPage, {
-      title: document && document.title || '',
-      referrer: lastPage,
-      path: nextPage
-    });
-  }
+const updateAnalyticsPage = (dispatch, lastPage, nextPage) => {
+  if (typeof document === 'undefined') return;
+  const name = document && document.title || '';
+  const properties = {
+    title: name,
+    referrer: lastPage,
+    path: nextPage
+  };
+  dispatch(segmentEventPage(name, null, properties));
```

Comment:
We no longer use window.analytics directly. We only use it through `redux-segment`. I think this is a good thing™.